pkgs/applications/virtualization/distrobox/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, lib, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "distrobox";
+  version = "1.2.13";
+
+  src = fetchurl {
+    url = "https://github.com/89luca89/distrobox/archive/refs/tags/1.2.13.tar.gz";
+    sha256 = "00x6rrw1z2z2w1aw2l2v773n25rd25d8fmvz45i188gjp316kyq1";
+  };
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  passthru.updateScript = ./update.sh;
+
+  installPhase = ''
+    runHook preInstall
+    install -D -m 0755 distrobox $out/bin/distrobox
+    install -D -m 0755 distrobox-export $out/bin/distrobox-export
+    install -D -m 0755 distrobox-create $out/bin/distrobox-create
+    install -D -m 0755 distrobox-init $out/bin/distrobox-init
+    install -D -m 0755 distrobox-list $out/bin/distrobox-list
+    install -D -m 0755 distrobox-rm $out/bin/distrobox-rm
+    install -D -m 0755 distrobox-enter $out/bin/distrobox-enter
+    install -D -m 0755 distrobox-enter $out/bin/distrobox-enter
```

Comment:
I think this block of `install`s and the following block would profit from a (common) `for` loop.

Thank you for your work! Distrobox looks like a useful addition :-)

Reply:
Hello! Thx for the review.
Could u elaborate on this for loop thing? Tho I think I got it, I'll try.

Reply:
It looks like that `installManPage` could help here.